CMakeLists.txt

```diff
@@ -313,17 +313,11 @@ endif(VALIDATE_APPDATA_FILE)
 if(TESTBUILD_OPENCL_PROGRAMS)
   set(TESTBUILD_OPENCL_PROGRAMS OFF)
 
-  macro(find_llvm versions)
-    foreach(version ${versions})
-      find_package(LLVM ${version} CONFIG)
-      if(LLVM_FOUND)
-        break()
-      endif()
-    endforeach()
-  endmacro()
+  find_package(LLVM CONFIG)
 
-  # 3.9 is the first version with which this works.
-  find_llvm("15;14;13;12;11;10;9;8;7;6.0;5.0;4.0;3.9")
+  if(${LLVM_PACKAGE_VERSION} LESS 7)
```

Comment:
should probably be VERSION_LESS instead of just LESS

Reply:
Should be simply `LLVM_VERSION`

Reply:
Testing that! Thanks.